packages/_vue3-migration-test/src/main.ts

```diff
@@ -36,10 +35,12 @@ createApp(App as Component)
   .use(xPlugin, {
     adapter,
     store,
+    initialXModules: [facetsXModule],
```

Comment:
You need it here because you miss register the `facets` XModule in each component: `clear-filters`, `facets`, `selected-filters-list` and `selected-filters`. In each of those components, there was a mixin to register `xComponentMixin(facetsXModule)` and now is needed `useRegisterXModule(facetsXModule);`

Registering it, you can avoid overriding here as initial ✌🏾